package/sbp_anpp_bridge/src/sbp_anpp_bridge.c

```diff
@@ -50,29 +51,67 @@
 #define SBP_MSG_ODOMETRY_FLAGS_SRC2 (0x08u)
 #define SBP_MSG_ODOMETRY_FLAGS_SRC3 (0x0Cu)
 
-bool anpp_debug = false;
+#define INVALID_TOW 0xffffffff
+#define TIME_SOURCE_MASK 0x07 /* Bits 0-2 */
+#define NO_TIME 0
+#define MS_IN_WEEK 7 * 24 * 60 * 60 * 1000
 
+bool anpp_debug = false;
```

Comment:
Maybe we can wrap these in a context and pass to them to callbacks instead of accessing as a bare global. Also could add debug as a setting in case we want dynamic troubleshooting capabilities

Reply:
yes that is a good idea, will do.